hq_superset/views.py

```diff
@@ -242,5 +246,11 @@ def select(self, hq_domain):
             )
             return redirect(url_for('SelectDomainView.list', next=request.url))
         response.set_cookie('hq_domain', hq_domain)
-        DomainSyncUtil(superset.appbuilder.sm).sync_domain_role(hq_domain)
+        if not DomainSyncUtil(superset.appbuilder.sm).sync_domain_role(hq_domain):
+            flash(
+                "You don't have the necessary HQ permissions to access this domain.",
```

Comment:
nit: might be useful to add the name of the domain to the message so the user knows which domain they were trying to access?

Reply:
Good idea!

Reply:
https://github.com/dimagi/commcare-analytics/pull/59/commits/f5384d1e50aa6c3b11a8b6913838e4168984c643